### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rishab1207**
Added How to use react-editor-js with react hooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,26 @@ render() {
...
+```
+
+#### Save Editor Data
+``` let data = await instanceRef.current.save();```
+
+#### Clear Editor
+``` instanceRef.current.clear(); ```
+
+#### Render Data inside the Editor
+``` instanceRef.current.render({ blocks }) ```
```
Comment (Jungwoo-An) [inline README.md:183]: Please use the code blocks. it improves readability.

**Review by Jungwoo-An — COMMENTED** [review]
@Rishab1207 Thanks for your interest! 👋

**Comment by Rishab1207** [comment]
Do I need to add something else in it?

**Comment by Jungwoo-An** [comment]
@Rishab1207 Looks nice. I'll merge this pr asap. Thanks! 🎉

File: README.md [hunk 2]
```diff
@@ -162,6 +162,32 @@ render() {
...
+
+#### Clear Editor
+```js 
+instanceRef.current.clear();
+```
+
+#### Render Data inside the Editor
+```js
+instanceRef.current.render({ blocks }) 
+```
```
Comment (Jungwoo-An) [inline README.md:189]:
```suggestion
```

Please delete this block or use enableReInitialize prop. I recommend delete block

File: README.md [hunk 3]
```diff
@@ -162,6 +162,32 @@ render() {
...
+
+const instanceRef = useRef(null); 
+
+<EditorJs tools={EDITOR_JS_TOOLS} instanceRef={(instance) => (instanceRef.current = instance)} />
+```
+
+#### Save Editor Data
+```js
+let data = await instanceRef.current.save();
+```
```
Comment (Jungwoo-An) [inline README.md:179]:
```suggestion
``` js
import { useRef } from React;

const instanceRef = useRef(null); 

async function handleSave() {
  const savedData = await instanceRef.current.save();
}

<EditorJs tools={EDITOR_JS_TOOLS} instanceRef={(instance) => (instanceRef.current = instance)} />
```
```

How about this?

**Review by Jungwoo-An — COMMENTED** [review]
I'm sorry that I didn't request at once. I think it would be great to change it a little bit. 👍

**Pull request closed without merging**